lls/main.c

```diff
@@ -224,13 +254,50 @@ lls_scan(__attribute__((unused)) struct rte_timer *timer, void *arg)
 		lls_cache_scan(lls_conf, &lls_conf->nd_cache);
 }
 
+/* Send gratuitous ARP or ND announcement to fulfill LACP TX requirement. */
 static void
+lls_lacp_announce(__attribute__((unused)) struct rte_timer *timer, void *arg)
```

Comment:
I had assumed that the TX function would fail if given if zero packets, but thanks for the idea to check that assumption! It actually works fine with transmitting zero packets, and both reduces the number of unnecessary packets and simplifies the code. I'll make the changes now.